### PR TITLE
added Operation Code

### PR DESCRIPTION
**Pull request by MasterKN48**
I have added Umbraco and OperationCode Hacktoberfest 2019 Swags. I tried to add details format according to Guide.
If there any mistake let me know.

### CODE REVIEW

**Comment by crweiner** [comment]
Hi @MasterKN48! Thank you for taking the time to add Umbraco and Operation Code to the Hacktoberfest Swag List!

Unfortunately, several devs at Umbraco don't seem to see the value of this little project and don't wish to be included here. Please see this post and the tweets I link to: https://github.com/crweiner/hacktoberfest-swag-list/issues/44#issuecomment-535131859

Personally, I don't understand it. Offering swag for Hacktoberfest is about marketing. Until they change their stance, I have agreed to not add Umbraco here for 2019. I also personally don't support companies that want to be close-minded about Hacktoberfest. 

If you feel similarly, you can certainly reach out to them on Twitter and let them know your thoughts.

So, please remove them from this particular PR and let me know when I can take another look at things. :+1:

File: README.md [hunk 1]
```diff
@@ -129,17 +151,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+    - Resolve 3 issues in any one of our repositories and receive a white or black Operation Code t-shirt!
+    - Merge 2 pull requests in any one of our repositories and receive a small pack of Operation Code stickers!
+- **How to sign up**: [Link](https://docs.google.com/forms/d/e/1FAIpQLSdZAC98DQS7DW1NJT0FrHYTI-88ST6_v8gMlILankqJG295DA/viewform) to claim swag.
+- **Notes**: See the [blog post](https://github.com/OperationCode/START_HERE) for more information.
 
-Nothing here yet!
-
-### 4 Merged PRs
-
-#### **Parity** (Limited Edition Parity Gym Bag)
```
Comment (crweiner) [inline README.md]: Any reason why you're deleting them??

Comment (MasterKN48) [reply README.md]: Sorry delete by mistake,
will fix it now.

File: README.md [hunk 2]
```diff
@@ -129,8 +151,13 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
-Nothing here yet!
+#### **OperationCode**
+- **Swag**: Stickers, T-Shirts
+- **Requirements**: 
+    - Resolve 3 issues in any one of our repositories and receive a white or black Operation Code t-shirt!
+    - Merge 2 pull requests in any one of our repositories and receive a small pack of Operation Code stickers!
+- **How to sign up**: [Link](https://docs.google.com/forms/d/e/1FAIpQLSdZAC98DQS7DW1NJT0FrHYTI-88ST6_v8gMlILankqJG295DA/viewform) to claim swag.
+- **Notes**: See the [blog post](https://github.com/OperationCode/START_HERE) for more information.
 
 ### 4 Merged PRs
```
Comment (MasterKN48) [inline README.md]: fix deleted data here

**Comment by crweiner** [comment]
> Should I remove Umbraco from list?

Yes, please remove Umbraco. If they only want to participate in half of Hacktoberfest, so be it.

File: README.md (hunk 2 above)
Comment (MasterKN48) [reply README.md]: removed umbraco from list

**Comment by MasterKN48** [comment]
I had remove umbraco from list.
If still any issue let me know.

**Comment by crweiner** [comment]
> I had remove umbraco from list.
> If still any issue let me know.

Hay again! If you could please review the mdl flags by [CodeClimate](https://codeclimate.com/github/crweiner/hacktoberfest-swag-list/pull/78) and resolve the conflicts, then I can get this merged today. Thank you!

**Comment by PatrickRNG** [comment]
Hi, i didn't see that there was already a PR for adding Operation Code, i just made one, since the issue for that was assigned to me yesterday..

**Comment by crweiner** [comment]
Hi @PatrickRNG! You're absolutely correct and I apologize for that slipping my mind. I've gotten over 150 emails today alone from my repos (most from here) due to Hacktoberfest contributions and you're correct that I assigned it to you yesterday. I'll have a look through your PR as well and can hopefully combine the best of both worlds with these two PRs. I appreciate your patience and for your time committing to this little project. :-)

**Pull request closed without merging**